tests/integration/cli/projects/test/tests/test_fixtures.py

```diff
@@ -13,7 +13,7 @@ def test_chain(chain):
 
 
 def test_contract(Contract):
-    assert isinstance(Contract, ContractInstance)
+    assert isinstance(Contract("0x0000000000000000000000000000000000000000"), ContractInstance)
```

Comment:
Technically, this won't be a contract
```suggestion
    assert isinstance(Contract("0x0000000000000000000000000000000000000000"), Address)
```